tests/dependencies.py

```diff
@@ -0,0 +1,12 @@
+import sys
+import pkg_resources
+
+
+def test_tensorflow_text_install():
```

Comment:
There is probably a better place for this to go, this isn't really a unit test so it shouldn't be in `tests`. I just added it here for now so I can see in the CI if it works on both Linux and on Windows.

Reply:
@wochinge When renaming `import_time.py` to `test_import_time.py` the test failed (average import time 0.9, while the test tries to assert <0.3). I undid the rename for now because it isn't related to this PR, but will create a new issue for this.